router-bridge/src/snapshots/router_bridge__plan__tests__it_works.snap

```diff
@@ -10,7 +10,8 @@ expression: "serde_json::to_string_pretty(&plan::<serde_json::Value>(Operational
       "kind": "Fetch",
       "serviceName": "accounts",
       "variableUsages": [],
-      "operation": "{me{name{first last}}}"
+      "operation": "{me{name{first last}}}",
+      "operationKind": "query"
     }
   }
 }
```

Comment:
It's worth noting that the `cargo insta review` also wanted me to add a:

```snap
assertion_line: 174
```

...directly under the `source: router-bridge/src/plan.rs` at the top of this `.snap` file (becoming line 3, and pushing the current line 3 down), but I did **not** include that:

![image](https://user-images.githubusercontent.com/841294/153866927-c0f81a6d-6b75-4589-b8ae-f9242e5aba56.png)